src/makeCalculator.js

```diff
@@ -38,6 +38,45 @@
  */
 function makeCalculator() {
   // write code here
-}
+  const calc = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+    multiply(value) {
+      this.result *= value;
+
+      return this;
+    },
+    divide(value) {
+      this.result /= value;
+
+      return this;
+    },
+
+    reset(value) {
```

Comment:
The 'reset' method should not have a parameter 'value' as it's not used. Please remove it. 🧹